### ISSUE TITLE
Implement Widget Narration as appropriate

### ISSUE THREAD

**Issue by immortius** [Wanderers-Of-The-Rift/wotr-mod#294]
This task is to implement widget narration as appropriate across WotR's UIs. There is a bit of investigation needed to determine what should be narrated, and an output should be some guidance for what is required narration-wise for future implementation.
